CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 The changelog lists most feature changes between each release. 
 
+## 2024-10-29
+
+- GTFS import: Mount local (modified) `download.sh` into `postgis-gtfs-importer` container.
+	- ⚠️ `$IPL_GTFS_IMPORTER_HOST_CUSTOM_SCRIPTS_DIR` must contain a `download.sh`, so if it doesn't, this is a breaking change!
```

Comment:
Why suddenly tabs?

Reply:
I don't know. 🤷 – Is it sudden? The `.editorconfig` doesn't define any rules for `.md` files.

Shall we change to 4 spaces?

Reply:
Yea, at least every file I know before is with spaces.